ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/BlockHeaderTestFixture.java

```diff
@@ -130,6 +133,11 @@ public BlockHeaderTestFixture baseFeePerGas(final long baseFee) {
     return this;
   }
 
+  public BlockHeaderTestFixture random(final Bytes32 random) {
+    this.random = Optional.of(random);
```

Comment:
Just in case. Are we sure that this field will never be null ?

Reply:
Yeah, `.ofNullable` would ensure tests that send a null random or tests that clone a blockheader with a null random don't throw.  I'll make it so